pythonforandroid/recipes/python3/patches/fix-ctypes-util-find-library.patch

```diff
@@ -10,9 +10,15 @@ diff --git a/Lib/ctypes/util.py b/Lib/ctypes/util.py
 +if True:
 +    def find_library(name):
 +        # Check the user app libs and system libraries directory:
-+        app_root = os.path.normpath(os.path.abspath('../../'))
-+        lib_search_dirs = [os.path.join(app_root, 'lib'), "/system/lib"]
-+        for lib_dir in lib_search_dirs:
++        from jnius import autoclass
++        activity = autoclass('org.kivy.android.PythonActivity').mActivity
++        package_name = activity.getApplicationContext().getPackageName()
++        manager = activity.getApplicationContext().getPackageManager()
++        nativeLibDir = manager.getApplicationInfo(
```

Comment:
super minor: use python naming convention `s/nativeLibDir/native_lib_dir/` I understand it's on the edge because we use java code, but still :pray: